simple_history/manager.py

```diff
@@ -263,6 +264,11 @@ def bulk_history_create(
                     for field in self.model.tracked_fields
                 },
             )
+
+            if custom_historical_attrs:
+                for name in custom_historical_attrs:
+                    setattr(row, name, custom_historical_attrs[name])
+
```

Comment:
This is a little bit shorter:
```suggestion
                **(custom_historical_attrs or {}),
            )
```